src/workerd/api/actor.c++

```diff
@@ -24,18 +24,18 @@ public:
     auto& context = IoContext::current();
 
     return context.getMetrics().wrapActorSubrequestClient(context.getSubrequest(
-        [&](SpanBuilder& span, IoChannelFactory& ioChannelFactory) {
-      if (span.isObserved()) {
-        span.setTag("actor_id"_kjc, kj::str(actorId));
+        [&](TraceContext& spans, IoChannelFactory& ioChannelFactory) {
```

Comment:
sorry for bikeshedding, but I think we can come up with better class/parameter name pair than this.

If we're going to keep `TraceContext`, then maybe `TraceContext& tracing`? (my vote)
If you like `spans`, then maybe `TraceSpans`?